tesseract_kinematics/src/kdl/kdl_inv_kin_chain_nr.cpp

```diff
@@ -93,7 +94,21 @@ IKSolutions KDLInvKinChainNR::calcInvKinHelper(const Eigen::Isometry3d& pose,
 
   KDLToEigen(kdl_solution, solution);
 
-  return { solution };
+  IKSolutions solution_set;
+  tesseract_kinematics::harmonizeTowardZero<double>(solution);
```

Comment:
Does this mean that the first solution will be the one that is closest to 0 rather than the one that is closest to the seed?

Reply:
That is correct. Should I change the logic so the original solution is first?

Reply:
Yeah I would think so. I'm not sure if we state it in the documentation, but my perception was that the first solution should be the closest to the seed. If you seed IK with the current location and take the first solution, I wouldn't want it to do a joint flip.

Reply:
Yea, I don't think we want to guarantee that just because it can be computationally intensive for the other IK solvers like opw_kinematics and ikfast where we would need to run a sort operation. The seed is more to help out the numerical solvers though for these we can add it first. Though I would push this on the user to perform the sort if needed to avoid a sort operation.